### ISSUE TITLE
Migration to SASS

### ISSUE THREAD

**Issue by 11joselu** [dalestone/angular2-gantt#24]
Hey!.
What do you think about migrating styles from css to sass?  it can be really great option.

I just [refactored a bit](https://github.com/11joselu/angular2-gantt/tree/sass-migration) and I am ready to submit a pull request.

## To Do
+ Change inline styles by CSS classes
+ Created custom classes for activity status
+ Refactor IBarStyle interface and functions

**Comment by dalestone** [comment]
Sure submit a pull request and I will review. 

I only have the css and templates inline due to an issue I encountered when using it in another application that uses webpack, it didn't seem to be able to find the css or html files.

**Comment by 11joselu** [comment]
Maybe we can resolve that building Gantt project using existing WebPack setup. I'm searching for a solution :)

**Comment by dalestone** [comment]
Yeah I agree there needs to be a better build process for the lib as currently I just publish the raw files to npm
